### PR TITLE
docs: add grafana to greptimedb

### PR DESCRIPTION
**Pull request by nicecui**

### CODE REVIEW

File: docs/db-cloud-shared/clients/grafana-integration.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+GreptimeDB can be configured as a [Grafana data source](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/).
+
+## Prometheus
+
+Click the Add data source button and select Prometheus as the type.
```
Comment (xtang) [inline docs/db-cloud-shared/clients/grafana-integration.md:5]:
```suggestion
Click the `Add data source` button and select `Prometheus` as the type.
```

File: docs/db-cloud-shared/clients/grafana-integration.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Fill in Prometheus server URL in HTTP:
+
+- GreptimeCloud: `https://<host>/v1/prometheus?db=<dbname>`
+- Self-host GreptimeDB: Your own host address, for example: `http://localhost:4000/v1/prometheus?db=<dbname>`
+
+Click basic auth in the Auth section and fill in your GreptimeDB username and password in Basic Auth Details:
```
Comment (xtang) [inline docs/db-cloud-shared/clients/grafana-integration.md:12]:
```suggestion
Click `basic auth` in the Auth section and fill in your GreptimeDB username and password in Basic Auth Details:
```

File: docs/db-cloud-shared/clients/grafana-integration.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- User: `<username>`
+- Password: *Your GreptimeDB password*
+
+Then click Save & Test button to test the connection.
```
Comment (xtang) [inline docs/db-cloud-shared/clients/grafana-integration.md:17]:
```suggestion
Then click `Save & Test` button to test the connection.
```

File: docs/db-cloud-shared/clients/grafana-integration.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## MySQL
+
+Click the Add data source button and select MySQL as the type. Fill in the following information in MySQL Connection:
```
Comment (xtang) [inline docs/db-cloud-shared/clients/grafana-integration.md:21]:
```suggestion
Click the `Add data source` button and select MySQL as the type. Fill in the following information in MySQL Connection:
```

**Comment by xtang** [comment]
@nicecui I left some comments, and the rest looks good to me

**Review by sunng87 — APPROVED** [review]

**Review by xtang — APPROVED** [review]

**Pull request merged**